### PR TITLE
Add `simple-app` project and update `company-web-server` to use pooling

### PR DESCRIPTION
**Pull request by SanjulaGanepola**
#### `simple-app`:
- [x] Created very simply command line application to showcase various features (follows closely with what is in the documentation -> https://github.com/Mapepire-IBMi/Mapepire-IBMi.github.io/pull/9)
  | Demo Type             | Description                                          |
  | --------------------- | ---------------------------------------------------- |
  | `--sql`               | Execute a SQL query                                  |
  | `--prepareStatement`  | Execute a prepared SQL query with parameters         |
  | `--clCommand`         | Execute a CL command                                 |
  | `--paginatingResults` | Fetch a specific number of rows and paginate results |
  | `--pooling`           | Execute a SQL query from a job pool                  |
  | `--jdbcOptions`       | Execute a SQL query with a job having JDBC options   |

#### `company-web-server`
- [x] No longer hosting Swager UI on GitHub pages and instead added instructions to run locally
- [x] Switched to use pool instead of single job
- [x] Added setup and usage instructions
- [x] On top of `/connect`, `/tracing`, and `/disconnect`:
  | Method | Endpoint                | Description                                               |
  | ------ | ----------------------- | --------------------------------------------------------- |
  | `GET`  | `/departments`          | Get a list of departments                                 |
  | `GET`  | `/departments/{deptNo}` | Get a department by department number                     |
  | `GET`  | `/employees`            | Get a list of employees                                   |
  | `GET`  | `/employees/{empNo}`    | Get an employee by employee number                        |
  | `GET`  | `/sales`                | Get a list of sales                                       |
  | `GET`  | `/sales/{salesPerson}`  | Get a list of sales associated with a sales person        |

### CODE REVIEW

**Review by jonnyz32 — COMMENTED** [review]
Are we intending to include the swagger/dist folder?

File: java/simple-app/LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (jonnyz32) [inline java/simple-app/LICENSE]:
```suggestion
   Copyright 2024 IBM CORPORATION
```

File: java/simple-app/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+1. Ensure the [Mapepire Server](https://mapepire-ibmi.github.io/guides/sysadmin) is installed and running on the IBM i.
+
+2. Clone the repository and build the application:
+
+    ```sh
+    git clone https://github.com/Mapepire-IBMi/samples.git
+    cd java/simple-app
+    mvn clean package
+    ```
```
Comment (jonnyz32) [inline java/simple-app/README.md:26]:
```suggestion
    ```
3. Copy and fill out the configuration properties
    ``` cp config.properties.sample config.properties```    
```

**Review by jonnyz32 — APPROVED** [review]

**Comment by SanjulaGanepola** [comment]
> Are we intending to include the swagger/dist folder?

Yes, running the swagger UI is part of the sample so that you can test out the endpoints.

**Pull request merged**